conf/evolutions/default/11.sql

```diff
@@ -0,0 +1,9 @@
+# --- !Ups
+
+ALTER TABLE forks ADD COLUMN issue_date timestamp;
+ALTER TABLE forks ADD COLUMN seconds_until_fork integer;
```

Comment:
is `integer` big enough?

Reply:
Yes, it should be enough. It represents the time between the fork time and the issue date. We can monitor this and increase it if the case.